### DISCUSSION TITLE
unexpected_permanent_failure error during vercel build

### DISCUSSION THREAD

**Discussion by abelchee** [vercel/community#2810] [Help]
### Summary

My deployment URL: https://vercel.com/abelchee/bd-web/8Mkmgq6LNqyj8nXjd1By5CvUQBzv

Error:
```
unexpected_permanent_failure: An unexpected error happened when running this build. We have been notified of the problem. If you have any questions, please contact Vercel Support https://vercel.com/help.
```

### Example

https://vercel.com/abelchee/bd-web/8Mkmgq6LNqyj8nXjd1By5CvUQBzv

### Steps to Reproduce

Keep failing on same commit

**Comment by swarnava** [comment]
Hi,

Can you try reducing number of static paths and give it a try?
